x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -33,8 +33,10 @@ import { SummarySection } from './dashboard_sections/summary_section';
 import { BenchmarksSection } from './dashboard_sections/benchmarks_section';
 import { CSPM_POLICY_TEMPLATE, KSPM_POLICY_TEMPLATE } from '../../../common/constants';
 
+type PosturePolicyTemplates = Extract<PosturePolicyTemplate, 'kspm' | 'cspm'>;
+
 const noDataOptions: Record<
-  PosturePolicyTemplate,
+  PosturePolicyTemplates,
```

Comment:
There are a lot of places in the dashboard that only expect `'kspm` | 'cspm'` as well, since the dashboard do not have a `vuln_mgmt` display. this change here might need to be broader.

I think the easiest solution might be to create two types
```typescript
type PosturePolicyTemaplte = 'kspm' | 'cspm' // stays the same
type CloudSecurityPolicyTemplate = PosturePolicyTemaplte | 'vuln_mgmt'
```